Erica/collaboration (#43)
UWB-ACM/Hackathon-Docs-2020: _docs/collaboration.md

## Collaborative Coding
To keep track of your code and prevent issues utilize the following:
- [Git](git-scm.com) = Track changes and sync your code with others
- [GitHub](Github.com) = Keep all your coding files in sync and help track changes in a Public or Private repo.

> *Note:* Github has something called "Projects" which is very much like Trello boards. So you can take advantage of these features!

Reviewer: Using spaces to keep line spacing for bulleted items consistent (unsure if it'll matter too much but it's a good idea anyhow)
```suggestion
  *Note:* Github has something called "Projects" which is very much like Trello boards. So you can take advantage of these features!
```

- [BitBucket.com](BitBucket.com) = Much like GitHub, but with private repositories.

## Collaborative Cloud
Check out [our guide on choosing a cloud provider](https://docs.uwbhacks.com/cloud_setup.html) to pick a cloud platform to use for your project.

### Sharing an Account

> If your team needs to work on a single Cloud account, you can choose to share your administrator credentials; however, this is not reccommended. Even though you may be working with friends, sharing the root admin account allows for anyone to see the credit card information on file, and additional sensitive materials. As such, it is advised that you create user accounts and give them specific permissions.

Reviewer: A few wording changes
```suggestion
If your team needs to work on a single cloud account, you can choose to share your administrator credentials; however, this is not recommended. Even though you may be working with friends, sharing the root admin account allows for anyone with access to view sensitive materials, grant additional users administrative privileges, and change the password for the root account, among other nefarious actions. As such, it is advised that you create new accounts in provider's identity management service and give the new user more limited permissions.
```

When you're ready, you can add users and give them permissions to use account services for your project.   When creating user accounts, you can follow the instructions for [AWS](https://docs.aws.amazon.com/marketplace/latest/userguide/marketplace-management-portal-user-access.html), [GCP](https://cloud.google.com/appengine/docs/standard/go/access-control#developers), and [Azure](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/add-users-azure-active-directory).

## Video Conferencing
As the UWB Hacks Spring 2020 hackathon is virtual this year, here are some options for video conferencing that can be used:

#### Discord

> A free software that allows for the creation of seperate groups, direct messaging, screen sharing, and group chats. Easy and casual vibe, that is quick to set up. This software retains a history of messages.

Reviewer: ```suggestion
Discord is free software for text and video chats that allows for the creation of separate groups (aka servers), direct messaging, screen sharing, and group chats. It has an easy and casual vibe that is quick to set up. This software retains a history of messages.
```

> [Get Discord here!](https://discordapp.com/)

Reviewer: Plug the event server one more time
```suggestion
[Get Discord here](https://discordapp.com/) and [join our event server here](https://discord.gg/EDBbxmt).
```

#### Zoom

> A software that that allows for the creation of meeting rooms, direct messaging, screen sharing, and whiteboards to draw notes on. It can be good for giving one-sided speeches, or group conversations. Zoom Pro is free for UWB students. This software does not retain a history of your messages.

Reviewer: ```suggestion
Zoom is videoconferencing software that that allows for the creation of meeting rooms, direct messaging, screen sharing, and whiteboards to draw notes on. It can be good for giving one-sided speeches, or group conversations. Zoom Pro is free for UWB students. This software does not retain a history of your messages.
```

[Get Zoom here!](https://zoom.us/)
#### Slack

> A software that is free for small groups, allows for seperate channels, and group video chat. Clean and accessable design, which allows for clear seperation between group chats. This software retains a history of messages.

Reviewer: ```suggestion
Slack is messaging software that is free for small groups, allows for separate channels, and group video chat. Clean and accessible design, which allows for clear separation between group chats. This software retains a history of messages.
```

> Zoom is videoconferencing software that that allows for the creation of meeting rooms, direct messaging, screen sharing, and whiteboards to draw notes on. It can be good for giving one-sided speeches, or group conversations. Zoom Pro is free for UWB students. This software does not retain a history of your messages.

Author: ```suggestion
Zoom is a videoconferencing software that that allows for the creation of meeting rooms, direct messaging, screen sharing, and whiteboards to draw notes on. It can be good for giving one-sided speeches, or group conversations. Zoom Pro is free for UWB students. This software does not retain a history of your messages.
```
